### ISSUE TITLE
DAC 19 - Medium Priority WCAG Level AA - Colour Contrast [Non-Text] 06

### ISSUE THREAD

**Issue by JadeneAd** [ONSvisual/income-scrolly#19]
<img width="373" alt="DAC_Colour_Contrast_Non_Text_06" src="https://user-images.githubusercontent.com/75850106/155561033-7a1b132a-fb64-41f4-b67f-8c1068f2b253.png">

### Issue
Low vision users may be unable to see the grey ‘Average’ circle of the diagram beneath the heading ‘Local extremes’. This is because the colour contrast of the circle against adjacent colours is below the 3:1 average. This applies to any other diagrams on the page that have the same colour usage.

Foreground: #808080 
Background: #CCCCCC
The contrast ratio is: 2.5:1 
1.4.11 Non-text Contrast (AA)
Fail for UI components and graphical objects

<img width="600" alt="DAC 19 Current code ref" src="https://user-images.githubusercontent.com/75850106/155561272-247733b2-7a17-4c48-92fb-fbed9c1cb80b.png">

**Low vision comments:**
 “When looking at the chart, I found the dark grey dot on the light grey bar to be inaccessible due to the text colour contrast, this fails at an approximate ratio of 2.5:1 as ideally this should be above 3:1”

### Suggested Solution
The colour contrast of non-text element against adjacent colours must be a minimum of 3:1.
